### PR TITLE
Add elasticsearch sink 3.0.1.4 docs

### PR DESCRIPTION
**Pull request by RobertIndie**

### CODE REVIEW

File: connectors/elasticsearch-sink/3.0.1.4/elasticsearch-sink.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+The [Elasticsearch](https://www.elastic.co/elasticsearch/) sink connector pulls messages from Pulsar topics and persists the messages to indexes. For more information about connectors, see [Connector Overview](https://docs.streamnative.io/docs/connector-overview).
+
+![](/images/connectors/elasticsearch-sink.png)
+
+This document introduces how to get started with creating an Elasticsearch sink connector and get it up and running.
+## Quick start
+
+### Prerequisites
+
+The prerequisites for connecting an Elasticsearch sink connector to external systems include:
```
Comment (shibd) [inline connectors/elasticsearch-sink/3.0.1.4/elasticsearch-sink.md:34]: What versions do we support? I think need to add this information. You can refer to:

https://github.com/apache/pulsar/blob/b7eab9469177eda2c56e36bb9871aab48a17d4ec/pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchConfig.java#L273-L282

File: connectors/elasticsearch-sink/3.0.1.4/elasticsearch-sink.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+Using this way requires you to download a **NAR** package to create a connector. You can download the version you need from the `download button` at the beginning of the article.
+- [Create a Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
+Using this way requires you to set the docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-elastic-search).
+
+No matter how you create an Elasticsearch sink connector, the minimum connector configuration for Elasticsearch 6.2+ contains the following parameters:
+```yaml
+configs:
+  elasticSearchUrl: "http://localhost:9200"
+  indexName: "my_index"
+  username: "scooby"
```
Comment (shibd) [inline connectors/elasticsearch-sink/3.0.1.4/elasticsearch-sink.md:58]: Is this the default account?

**Comment by shibd** [comment]
hi @RobertIndie, I think the folder name needs to keep a two-digit version. Like kinesis connector:  https://github.com/streamnative/pulsar-hub/tree/master/connectors/kinesis-sink/3.1

**Comment by RobertIndie** [comment]
> I think the folder name needs to keep a two-digit version. Like kinesis connector: https://github.com/streamnative/pulsar-hub/tree/master/connectors/kinesis-sink/3.1

@shibd , The connector hub can turn the 4-digit version to the 2-digit version automatically. Is there any other reason that we need to use 2 digits?

**Review by shibd — APPROVED** [review]

**Pull request merged**